pkg/filter/dt/filter_mysql_prepare.go

```diff
@@ -178,11 +189,13 @@ func (f *_mysqlFilter) processAfterPrepareUpdate(ctx context.Context, conn *driv
 
 func (f *_mysqlFilter) processSelectForPrepareUpdate(ctx context.Context, conn *driver.BackendConnection,
 	result proto.Result, stmt *proto.Stmt, selectStmt *ast.SelectStmt) error {
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_select_for_update")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.Executable`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "executable")
```